layouts/downloads/list.html

```diff
@@ -15,6 +15,11 @@ <h2 >Decred Wallet For Mobile</h2>
         <a href="https://play.google.com/store/apps/details?id=com.decred.dcrandroid.mainnet" target="_blank"><img src="{{ "../images/google-store-icon.png" | absURL }}" alt="AndroidApp"></a>
         <a href="https://apps.apple.com/us/app/decred-wallet/id1462247643" target="_blank"><img src="{{ "../images/apple-store-icon.png" | absURL }}" alt="iOSAPP"></a>
     </div>
+
+    <div class="downloads-container">
+        <h2 >Decred Wallet For Computers</h2>
+        <a href="https://github.com/planetdecred/godcr/releases" target="_blank">Download GoDCR binaries</a>
```

Comment:
![Screenshot from 2022-05-18 22-48-29](https://user-images.githubusercontent.com/25265396/169161516-e961ae3d-ffc0-4ca1-a7a1-ed8881ad20ee.png)

Let's have download buttons for individual platforms, so they can download directly

something like in the picture above